docs/nasa_exoplanet_archive/nasa_exoplanet_archive.rst

```diff
@@ -6,56 +6,168 @@
+2. The list of confirmed planets discovered by TESS and their coordinates:
 
 .. code-block:: python
 
-        >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
-        >>> hatp11b = NasaExoplanetArchive.query_planet('HAT-P-11 b')
+    >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
+    >>> NasaExoplanetArchive.query_criteria(table="exoplanets", select="pl_hostname,ra,dec",
+    ...                                     where="pl_facility like '%TESS%'")
+    <QTable length=45>
+    pl_hostname     ra        dec           sky_coord
+                   deg        deg            deg,deg
+       str11     float64    float64           object
+    ----------- ---------- ---------- ---------------------
+       HD 39091  84.291214 -80.469124  84.291214,-80.469124
+      HD 219666 349.556792 -56.903786 349.556792,-56.903786
+            ...        ...        ...        ...        ...
+      HD 221416  353.03363 -21.801424  353.03363,-21.801424
+        TOI-150 112.965667 -73.606153 112.965667,-73.606153
+
 
+3. The list of confirmed planets discovered using microlensing that have data available in the archive:
+
+.. code-block:: python
 
-Properties of a particular planet
-=================================
+    >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
+    >>> NasaExoplanetArchive.query_criteria(
+    ...     table="exoplanets", where="pl_discmethod like 'Microlensing' and st_nts > 0")
+    <QTable masked=True length=32>
+         pl_hostname      pl_letter ... pl_facility       sky_coord
+                                    ...                    deg,deg
+            str21            str1   ...     str4            object
+    --------------------- --------- ... ----------- ---------------------
+        MOA-2011-BLG-028L         b ...         MOA    270.854,-29.213417
+      OGLE-2012-BLG-0724L         b ...        OGLE 268.968292,-29.818528
+                      ...       ... ...         ...                   ...
+        MOA-2011-BLG-262L         b ...         MOA 270.097833,-31.245258
+      OGLE-2015-BLG-0966L         b ...        OGLE  268.75425,-29.047111
 
-The properties of each planet are stored in a table, with `columns defined by
-the NASA Explanet Archive <https://exoplanetarchive.ipac.caltech.edu/docs/API_exoplanet_columns.html>`_.
-There is also a special column of sky coordinates for each target, named
-``sky_coord``.
+
+4. The list of confirmed planets where the host star name starts with "Kepler" using a *wildcard search*:
 
 .. code-block:: python
 
-        >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
-        >>> hatp11b = NasaExoplanetArchive.query_planet('HAT-P-11 b')
+    >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
+    >>> NasaExoplanetArchive.query_criteria(
+    ...     table="exoplanets", where="pl_hostname like 'Kepler%'", order="pl_hostname")
+    <QTable masked=True length=2312>
```

Comment:
some of these table have already changed (we don't yet remote doctest, so it don't yet cause troubles), so no action items for now, but maybe the future the examples could be updated to be roboust for server the changing upstream updates

Reply:
I took a look at this and it would be somewhat difficult to construct example queries that demonstrate the full range of options available without changing with the server data. This is because all the most interesting queries *depend* on the state of the data. If this really is a showstopper, we could move the discussion of these features to the text rather than examples, but I think that would make the docs way less useful. Your call!

Reply:
Now, this is definitely not a showstopper, especially not until we get around and fix the doctesting infra. And then we might just limit the testing that everything runs, with ignoring matching the exact output.